### PR TITLE
Index toc rendering issue

### PR DESCRIPTION
**Pull request by Roiocam**
Fixes https://github.com/apache/incubator-pekko/issues/975#issuecomment-1896210127

We can remove those element not belong the selected group.

still draft because this is not finish yet.

### CODE REVIEW

**Comment by Roiocam** [comment]
This PR hides the element when toc header link is not the current group, I have verified this on my laptop.

But after PR #86 and #84, this plugin won't render the navigator anymore, because we put these render scripts to CDN,

you could verify this PR following this way:

1. checkout to the latest pekko project
2. nothing to do and then clean up and `docs/paradoxBrowse`, which means compile doc with the paradox
3. use this PR script to replace `docs/target/paradox/site/main/assets/javascripts/groups.js`
4. open the browser and refresh pekko doc page

<details><summary>Select Current Group to Java</summary>

截屏2024-01-24 10 30 24


</details> 

<details><summary>Select Current Group to Scala</summary>

截屏2024-01-24 10 30 27


</details>

**Comment by Roiocam** [comment]
cc @He-Pin @mdedetrich @pjfanning @samueleresca

File: theme/src/main/assets/assets/javascripts/groups.js [hunk 1]
```diff
@@ -148,6 +148,41 @@ $(function() {
       });
     });
 
+     // Index Page toc
+     $(".toc").each(function() {
+      var tocRoot = $(this);
+      function findHeaderElements(element) {
+        var headerElements = element.find(".header");
+        headerElements.each(function() {
```
Comment (He-Pin) [inline theme/src/main/assets/assets/javascripts/groups.js:156]: with a filter? and then foreach?

**Review by He-Pin — APPROVED** [review]
thanks, lgtm, but there is conflict. @Roiocam

File: theme/src/main/assets/assets/javascripts/groups.js (hunk 1 above)
Comment (Roiocam) [reply theme/src/main/assets/assets/javascripts/groups.js:156]: Yes, that is simpler, I will try it later

Comment (He-Pin) [reply theme/src/main/assets/assets/javascripts/groups.js:156]: This fix the `blank` entry, it should be fixed in `css` but fixed in `js`. @mdedetrich @pjfanning @samueleresca
and later should update https://github.com/apache/incubator-pekko-site

**Comment by Roiocam** [comment]
close and reopen in the right repo: https://github.com/apache/incubator-pekko-site/pull/72

**Pull request closed without merging**